firebase-perf/src/main/java/com/google/firebase/perf/metrics/FrameMetricsCalculator.java

```diff
@@ -49,6 +49,19 @@ public int getSlowFrames() {
     public int getTotalFrames() {
       return totalFrames;
     }
+
+    /**
+     * Subtracts frame-time counts of the argument object (that) from the current object (this).
+     *
+     * @param that the subtrahend PerfFrameMetrics object.
+     * @return difference of this and the argument.
+     */
+    public PerfFrameMetrics subtract(PerfFrameMetrics that) {
```

Comment:
I still think that this method needs to be renamed to something like `deltaFrameMetricsFromSnapshot(PerfFrameMetrics that)`.